### PR TITLE
fix: Remove always nil return value

### PR DESCRIPTION
**Pull request by erezrokah**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

New linter caught this. I'm doing this in a separate PR to make verifying the change easier

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: pkg/module/drift/config_parser.go [hunk 1]
```diff
@@ -175,7 +175,6 @@ func (p *Parser) interpret(cfg *BaseConfig) hcl.Diagnostics {
 			}
 		}
 	}
-	return nil
```
Comment (erezrokah) [inline pkg/module/drift/config_parser.go:178]: Return value is always `nil` and will actually cause a panic in callers I believe

Comment (disq) [reply pkg/module/drift/config_parser.go:178]: https://go.dev/play/p/JxmTqiIB3Wj?v=goprev

Comment (erezrokah) [reply pkg/module/drift/config_parser.go:178]: Cool, thanks for the context. Should we still remove the return value? It's not really used

Comment (disq) [reply pkg/module/drift/config_parser.go:178]: Sure.

**Review by disq — APPROVED** [review]

**Pull request merged**